lib/CommandDeclarations.ts

```diff
@@ -761,6 +762,15 @@ export const Commands: Command[] = [
             },
         ],
     },
+    {
+        aliases: ['stock', 'stonk'],
+        primaryCommand: {
+            argsFormat: Args.Split,
+            implementation: handleStock,
+            description: 'Check a stock price.',
+            needDb: false,
```

Comment:
Can we add an example with a ticker being given. The default generated help does not include a ticker value.